raiden/tests/integration/conftest.py

```diff
@@ -0,0 +1,42 @@
+def pytest_addoption(parser):
+
+    parser.addoption(
+        '--local-matrix',
+        action='store',
+        dest='local_matrix',
+        default=None,
+        help=(
+            'Path to script/command that starts a local matrix server. If given, '
+            'the integration tests will run using matrix instead of udp.'
+        )
+    )
+
+    parser.addoption(
+        '--matrix-host',
+        action='store',
+        dest='matrix_host',
+        default='localhost',
+        help="Host name of local matrix server if used, default: 'localhost'"
```

Comment:
Question on pytest parser `addoption`. Where can we see this help string? Doing `pytest --help` does not seem to display these arguments. I just wanted to check if specifying the default manually is needed or if the help string does that for you.

Reply:
`pytest --help` displays the help strings when run in the `integration` folder, and it does not display the default values by itself...